test/base/resources/cloud_event.go

```diff
@@ -21,6 +21,8 @@ type CloudEvent struct {
 	ID       string
 	Type     string
 	Source   string
+	ExtensionName string
```

Comment:
I think this should be a `map[string]string`, so that we are allowed to set any number of extensions.

Like: 

`Extensions map[string]string`

and you can remove `ExtensionName` and `ExtensionValue`

Reply:
@Fredy-Z are there any plans of using the CloudEvents sdk instead of this CloudEvent struct?

Reply:
Do you mean https://github.com/cloudevents/sdk-go/blob/master/pkg/cloudevents/eventcontext_v03.go#L21? We can replace it now, but can be in a following PR.

Reply:
I agree it should `map[string]interface{}`, and also we need to add a test case that uses multiple extensions. You can pass the map extension as a `json` to the `sendevents` image.

Reply:
let's do it in a follow up PR... Yeah, I guess I was talking about just removing the cloud_event.go and use the types in the sdk, the cloudevents.Event, and its context. But that can be done in the future